Jellyfin.Plugin.Tvdb/Providers/TvdbMissingEpisodeProvider.cs

```diff
@@ -125,16 +135,20 @@ private void OnProviderManagerRefreshComplete(object? sender, GenericEventArgs<B
         {
             if (!IsEnabledForLibrary(genericEventArgs.Argument))
             {
+                _logger.LogDebug("{ProviderName} not enabled for {InputName}", ProviderName, genericEventArgs.Argument.Name);
                 return;
             }
 
+            _logger.LogDebug(nameof(OnProviderManagerRefreshComplete) + ": Try Refreshing for Item {Name} {Type}", genericEventArgs.Argument.Name, genericEventArgs.Argument.GetType());
```

Comment:
These should also be moved to templated logging

Reply:
I will do so. it was used like that by intention. the templating has in most cases a value for structured logger, to have the information in an extra field. But the method name is not so valuable here to treat it like that. But anyway, at the end it is logged in the message.